pilot/pkg/networking/util/util.go

```diff
@@ -44,7 +44,9 @@ const (
 	PassthroughCluster = "PassthroughCluster"
 	// SniClusterFilter is the name of the sni_cluster envoy filter
 	SniClusterFilter = "envoy.filters.network.sni_cluster"
-
+	// IstioMetadataKey is the key under which metadata is added to a route or cluster
```

Comment:
Can you also update https://github.com/istio/istio/blob/master/pilot/pkg/proxy/envoy/v2/eds.go#L161 to use this constant?

Reply:
Done.